promote-to-stable

```diff
@@ -45,29 +45,34 @@ task :promote_artifacts_to_stable do
 end
 
 task :update_addons_metadata do
-  path_without_bucket_name = S3_ADDONS_STABLE_BUCKET.sub(%r{^[^/]+/}, '')
+  path_without_stable_bucket_name = S3_ADDONS_STABLE_BUCKET.sub(%r{^[^/]+/}, '')
+  path_without_experimental_bucket_name = S3_ADDONS_EXPERIMENTAL_BUCKET.sub(%r{^[^/]+/}, '')
 
   pg_addon_metadata = AddonMetadata.new(
-    addon_file_s3_parent:  "#{path_without_bucket_name}/#{go_full_version}",
+    addon_file_s3_parent:  "#{path_without_stable_bucket_name}/#{go_full_version}",
     metadata_full_s3_path: "s3://#{S3_ADDONS_STABLE_BUCKET}/go-postgresql.json",
     version:               go_full_version,
     prefix:                'pg'
   )
 
   bc_addon_metadata = AddonMetadata.new(
-    addon_file_s3_parent:  "#{path_without_bucket_name}/#{go_full_version}",
+    addon_file_s3_parent:  "#{path_without_stable_bucket_name}/#{go_full_version}",
     metadata_full_s3_path: "s3://#{S3_ADDONS_STABLE_BUCKET}/go-business-continuity.json",
     version:               go_full_version,
     prefix:                'bc'
   )
 
   pg_metadata_file = Tempfile.new(['pg_metadata', '.json'], Dir.tmpdir)
   sh("AWS_PROFILE=extensions aws s3 cp s3://#{S3_ADDONS_EXPERIMENTAL_BUCKET}/#{go_full_version}/pg_metadata.json #{pg_metadata_file.path}")
-  pg_addon_metadata.append_to_existing(JSON.parse(File.read(pg_metadata_file.path)))
+  new_pg_addon_metadata = JSON.parse(File.read(pg_metadata_file.path))
+  new_pg_addon_metadata['location'].sub!(/^#{path_without_experimental_bucket_name}/, path_without_stable_bucket_name)
+  pg_addon_metadata.append_to_existing(new_pg_addon_metadata)
 
   bc_metadata_file = Tempfile.new(['bc_metadata', '.json'], Dir.tmpdir)
   sh("AWS_PROFILE=extensions aws s3 cp s3://#{S3_ADDONS_EXPERIMENTAL_BUCKET}/#{go_full_version}/bc_metadata.json #{bc_metadata_file.path}")
-  bc_addon_metadata.append_to_existing(JSON.parse(File.read(bc_metadata_file.path)))
+  new_bc_addon_metadata = JSON.parse(File.read(bc_metadata_file.path))
+  new_bc_addon_metadata['location'].sub!(/^#{path_without_experimental_bucket_name}/, path_without_stable_bucket_name)
+  bc_addon_metadata.append_to_existing(new_bc_addon_metadata)
```

Comment:
Something like that, yes. Except there is no need to have the method path_without_bucket_name be called twice, I think.